tests/foreman/cli/test_contentaccess.py

```diff
@@ -38,42 +41,48 @@
 
 
 @pytest.fixture(scope='module')
-def module_lce(module_sca_manifest_org):
-    return entities.LifecycleEnvironment(organization=module_sca_manifest_org).create()
-
+def rh_repo_setup_ak(module_sca_manifest_org, module_target_sat):
+    """Use module sca manifest org, creates rhst repo & syncs it,
+    also create CV, LCE & AK and return AK"""
+    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_sca_manifest_org.id,
+        product=PRDS['rhel'],
```

Comment:
Should we be using rhel7 repos here? or does it not matter?

Reply:
`PRDS['rhel']` is product name and `rhel7` is not present in [constants](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/constants/__init__.py#L269)